Problems with Archlinux

# my configuration :
* an up to date Archlinux (`Linux archsuizokukan 4.2.5-1-ARCH #1 SMP PREEMPT Tue Oct 27 08:13:28 CET 2015 x86_64 GNU/Linux`)
* alure is installed (a `$pacman -Ss alure` returns `community/alure 1.2-4 [installed]`)
* a fresh installation of mxe : `git clone https://github.com/mxe/mxe`, `cd mxe`, then `make MXE_TARGETS='x86_64-w64-mingw32.static'`

# the error :
`make MXE_TARGETS='x86_64-w64-mingw32.static'` compiles a lot of stuff (mxe-conf, gcc... zlib, ...) and exits returning the following message :
> Failed to build package alure for target x86_64-w64-mingw32.static!

# the logs from log/alure_x86_64-w64-mingw32.static :
> [ 92%] Building CXX object CMakeFiles/ALURE32-static.dir/src/codec_mpg123.obj
> /home/suizokukan/mxe20151229/usr/bin/x86_64-w64-mingw32.static-g++   -DDYNLOAD=1 -DHAVE_CONFIG_H -D_FILE_OFFSET_BITS=64 -D_WIN32 -DAL_LIBTYPE_STATIC -DALURE_STATIC_LIBRARY -O3 -DNDEBUG @CMakeFiles/ALURE32-static.dir/includes_CXX.rsp   -Wextra -Wall -funswitch-loops  -DALURE_STATIC_LIBRARY -o CMakeFiles/ALURE32-static.dir/src/codec_mpg123.obj -c /home/suizokukan/mxe20151229/tmp-alure-x86_64-w64-mingw32.static/alure-1.2/src/codec_mpg123.cpp
> /home/suizokukan/mxe20151229/tmp-alure-x86_64-w64-mingw32.static/alure-1.2/src/codec_mpg123.cpp:33:20: fatal error: mpg123.h: No such file or directory
> #include <mpg123.h>

Any help would be appreciated !

I tried #1110 (namely, `$(PKG)_DEPS     := gcc flac libsndfile mpg123 ogg openal vorbis` in src/alure.mk) :

mpg123 was correctly compiled but now, there's an error about a missing file, namely `fluidsynth.h` .
details here : https://gist.github.com/suizokukan/c8c02be3beefcda06bb9

Naively, I tried to fix src/alure.mk this way : `$(PKG)_DEPS     := gcc flac libsndfile mpg123 fluidsynth ogg openal vorbis` ... but I got the following error : 

> make: ***  Aucune règle pour fabriquer la cible « /home/suizokukan/mxe/usr/x86_64-w64-mingw32.static/installed/fluidsynth »
(id est : make: *** No rule to create the target /home/suizokukan/mxe/usr/x86_64-w64-mingw32.static/installed/fluidsynth)

--- a/src/alure.mk
+++ b/src/alure.mk
@@ -8,7 +8,7 @@ $(PKG)_CHECKSUM := 465e6adae68927be3a023903764662d64404e40c4c152d160e3a8838b1d70
 $(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.bz2
 $(PKG)_URL      := http://kcat.strangesoft.net/alure-releases/$($(PKG)_FILE)
-$(PKG)_DEPS     := gcc flac libsndfile ogg openal vorbis
+$(PKG)_DEPS     := gcc flac libsndfile mpg123 ogg openal vorbis
 
 define $(PKG)_UPDATE
     $(WGET) -q -O- http://repo.or.cz/w/alure.git/tags | \

